.editorconfig

```diff
@@ -0,0 +1,13 @@
+# EditorConfig configuration for doldecomp/melee
+# https://EditorConfig.org
+
+# Top-most EditorConfig file
+root = true
+
+[*]
```

Comment:
Ia this going to be a problem for stuff like yml and gas?

Reply:
Not with the -disable-indent-size option. If we remove that, we'll need to reindent the asm files. Let me add a special case for the 2-space indentation of our .yml files.

Reply:
TBH, there's so much mixed spaces/tabs in the asm that I might just specify in the .editorconfig that either is fine in .s files. LMK what you think

Reply:
I think we should just replace all tabs with spaces. Either in a separate PR (which merges before this one) or in this PR, I don't really care.

Reply:
Sounds good. I'll issue a separate PR with the whitespace changes.